data/json/snippets/snippets.json

```diff
@@ -1803,6 +1803,54 @@
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["Dear John,\nI want you to know I intended to wait until you returned from Korea, but as time has gone it’s become to be more than I can take. From what they’re saying on the news it seems you’ll be gone longer than was first thought. I sometimes stay up at night wondering if you will ever make it home at all and it’s just so much to think about.\nI'm sorry, but this just isn't working. I know you never chose to be drafted and that you would be by my side if you could. I want you to know that it’s not your fault but I’m at a time in my life where I need someone here to support me. I know you must be angry, but I know you’ll understand. I hope you get home safe John.\nSincerely,\nJen\""
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"WOLF ATTACKS ON THE RISE!\nIncreasing reports across the New England area of wolves attacking local wildlife, and even people. Forest rangers are baffled with the oddly aggressive behaviour, and while they have not yet determined the cause, they advise anyone sighting a wolf to avoid contact. If threatened - fight back. Running seems to provoke them to attack.\""
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"NEW LINE OF CBMS RELEASED!\nGE are proud to announce the latest line of CBMS, the Metabolic Exchange System! Now your batteries can be powered from the food you eat, no more do you have to rely on expensive energy prices! Available now - while stocks last!\n(GE is not responsible for starvation that may or may not be caused by this system. If symptoms of hunger or lower power persist after eating, please consult your local doctor)\""
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"VAC AND RIVTECH WEAPON MANUFACTURERS AT IT AGAIN!\nThe two biggest names in military and civilian firearms are back at their competitive antics. After a court meeting between the two companies that left VAC suing for “intellectual theft”, Rivtech was reportedly exposed in an attempt to sabotage a Virgil Armament Company manufacturing plant. It is unsure if Rivtech will be facing legal consequences, and both companies refuse to comment.\""
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"STRANGE EVENTS IN SMALL NEW ENGLAND TOWN\nAccording to residents of a small New England town, a local science laboratory has been keeping them up at nights. Many people spoken to complain about loud noises and flashes of light coming from a government sanction lab outside of town. Other eyewitnesses suggest seeing “crazed, naked men” in the woods around the lab. Government officials withheld any comment on these occurrences, suggesting that the noises and flashes of light are coming from military training routines. They refused to talk anymore when we mentioned the sightings of the crazed, naked men.\""
```

Comment:
Check punctuation, you've got a `“ ”` instead of a pair of ` \"`.
Looks like another news article, not note.

Reply:
Corrected